site/_posts/2018-01-29-auth-overview.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "ManageIQ Authentication Overview"
+author: jvlcek
+date: 2018-01-29
```

Comment:
You might want to date the Overview post before the Troubleshooting one (or on the same day is fine as well)

Reply:
@cybette I want the overview to be listed on the `all blogs` page above the Troubleshooting post. The simplest way I found to do this is to have the date of the Overview post be ahead of the Troubleshooting post.

Reply:
ah ok. you can add a timestamp e.g. `date: 2018-01-31 08:30:00`

better to use use different timestamps (which doesn't show on the blog post but will order them).